torch/_dynamo/external_utils.py

```diff
@@ -3,3 +3,7 @@
 
 def is_compiling():
     return False
+
+
+def is_exporting():
```

Comment:
NIT: can you add a docstring here explaining various use cases, like how user can skip over certain part of the code etc?